README.md

```diff
@@ -497,6 +497,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Webex](https://www.webex.com/) — Video meetings with a free plan offering 40 minutes per meeting with 100 attendees.
   * [RingCentral](https://www.ringcentral.com/) — Video meetings with a free plan offering 50 minutes per meeting with 100 participants.
   * [GitBook](https://www.gitbook.com/) — Platform for capturing and documenting technical knowledge — from product docs to internal knowledge bases and APIs. Free plan for individual developers.
+  * [DevToolLab](https://devtoollab.com) — A comprehensive platform for developers offering various tools like JSON/XML formatter, JSON Viewer, and more. Free tier available for developers.
```

Comment:
there is no information about this "free tier available for developers" on the site.  No pricing details at all.

Reply:
please checkout this : [DevToolLab pricing](https://devtoollab.com/pricing)

Reply:
OK, like other entries here please state what's free

Reply:
Done Please review it again Thanks